### PR TITLE
Add two sections to dryville_story.md

### PR DESCRIPTION
**Pull request by wildbio**
### I added two sections to dryville_story.md (see #8):
1. Getting Water to Your Homes
2. Dryville's First Water Works

### CODE REVIEW

**Review by jesse-ross — CHANGES_REQUESTED** [review]
This looks good! There is only one change I would request. In the Getting Water to Your Homes section, there should be a link for "water does flow downhill". Could you please add that?

**Review by jesse-ross — APPROVED** [review]
Oh, great, you caught another link, too! Looks good, feel free to merge 😻

File: dryville_story.md [hunk 1]
```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
...
+
+The problem is, the houses are higher than the creek, and water does not flow uphill. [Water does flow downhill](https://www.usgs.gov/special-topic/water-science-school/science/watersheds-and-drainage-basins), so you build a big water storage tank on a hill (hopefully at the highest point in town) and establish a water brigade to fill it full of water. You can run pipes from the storage tank down to everyone's home. The pipes go right to your faucet and you use gravity (it's free!) to get the water delivered. Works great -- just like it did in the big city you came from.
```
Comment (wildbio) [inline dryville_story.md:23]: Link added here

File: dryville_story.md [hunk 2]
```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
...
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the [creek](https://www.usgs.gov/special-topic/water-science-school/science/rivers-streams-and-creeks) uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outside world to bring in money to buy things with yet, you need to build your own pump. And you can't use electricity since you haven't built a power plant yet (but you will). Can you think of a source of energy that can run your pump? A windmill could do the job, but in the desert there isn't always wind. Your creek! The water in your creek is flowing nicely -- so you build a paddle wheel in the river. The paddle spins in the creek's current and turns a rod that runs your homemade pump. Voila! You've got water being pumped uphill into your storage tank where gravity lets it flow to your homes.
```
Comment (wildbio) [inline dryville_story.md:25]: Link added here

**Comment by wildbio** [comment]
Thanks, Jesse! I'm getting the message that there are no conflicts between this branch and the base branch, but, "Only those with write access to this repository can merge pull requests."

**Comment by jesse-ross** [comment]
Right! Merging now.

**Pull request merged**